auth/types/account_retriever.go

```diff
@@ -2,18 +2,28 @@ package types
 
 import (
 	"fmt"
+	cliContext "github.com/cosmos/cosmos-sdk/client/context"
```

Comment:
Could you add an empty line above to separate third-party imports from the built-in ones?

Reply:
Will do 👌